sklearn/tree/_criterion.pyx

```diff
@@ -553,29 +487,29 @@ cdef class Entropy(ClassificationCriterion):
         impurity_right : double pointer
             The memory address to save the impurity of the right node
         """
-        cdef SIZE_t* n_classes = self.n_classes
-        cdef double* sum_left = self.sum_left
-        cdef double* sum_right = self.sum_right
         cdef double entropy_left = 0.0
         cdef double entropy_right = 0.0
         cdef double count_k
+        # Pointeres benchmark better than using the memorview directly
```

Comment:
This is the only place where the strides are still needed. When I benchmark by using the memorview directly, I get a runtime regression.

Reply:
It's weird that this does not happen in other places where double-axes memory-views are used. 
```suggestion
        # Pointers are being used here because benchmarks
        # showed regression when using memory-views.
```